Makefile

```diff
@@ -56,7 +56,6 @@ lint-fix: sync-deps
 TESTARGS ?= test/
 test:
 	node_modules/.bin/jest --detectOpenHandles --colors $(TESTARGS)
-	make test-inspector
```

Comment:
Why is this removed?

Reply:
cause when you want to run the inspector test you use the `make test-inspector` command.
and when you want tu run the sdk/ecs test you use the `make test`. Just to save time and ignore the inspecrtor tests when you want to run a single test using the TESTARGS

Reply:
this is not being used by the github actions.
there we use the make test-cli, make test-inspector and make test-coverage.

Maybe we can rename it 🤔  like `make test-sdk` instead of just test